src/sidebars/search/search.html

```diff
@@ -1,6 +1,6 @@
 <div ng-keydown="searchCtrl.onSearchKeyDown($event)">
   <div>
-      <img src="assets/png/search_tab.png" class="top-icon" title="Search Stocks" draggable="false" />
+      <img src="assets/png/search_tab.png" ng-click="sidebarCtrl.toggleClick(); $event.stopPropagation();" class="top-icon" title="Search Stocks" draggable="false" />
```

Comment:
Actually this is necessary. Otherwise once the search is open, the click on the icon will be superseded by the click on search, so `searchClick` is called instead of `toggle`. This results in the icon no longer closing the search